scripts/prepare-environment-k3d.sh

```diff
@@ -51,10 +53,23 @@ check_dependency kubectl helm
 create_docker_pull_secret
 
 echo "Preparing Epinio manifest"
-sed -i "s/10.86.4.38.omg.howdoi.website/$EPINIO_SYSTEM_DOMAIN/" assets/installer/manifest.yaml
+echo "Replacing the system domain"
```

Comment:
These `sed` commands may be gone in the future if the installer supports variables.

Reply:
Yes, otherwise if this will be as is for some time it could be more "readable" to use something like `sed -i "s/%EPINIO_SYSTEM_DOMAIN%/$EPINIO_SYSTEM_DOMAIN/" assets/installer/manifest.yaml`